prefs.js

```diff
@@ -276,13 +276,150 @@ const EasyScreenCastSettingsWidget = new GObject.Class({
                 Settings.FILE_CONTAINER_SETTING_KEY, this.Ref_combobox_Container,
                 'active', Gio.SettingsBindFlags.DEFAULT);
 
-            //implements file container option
+            //implements file stack resolution
+            this.Ref_stack_FileResolution = builder.get_object(
+                'stk_FileResolution');
+
+            //implements file container resolution
+            this.Ref_stackswitcher_FileResolution = builder.get_object(
+                'sts_FileResolution');
+
+            //implements file resolution preset spinner
             this.Ref_combobox_Resolution = builder.get_object(
                 'cbt_FileResolution');
             Settings.settings.bind(
-                Settings.FILE_RESOLUTION_SETTING_KEY, this.Ref_combobox_Resolution,
+                Settings.FILE_RESOLUTION_TYPE_SETTING_KEY,
+                this.Ref_combobox_Resolution,
                 'active', Gio.SettingsBindFlags.DEFAULT);
 
+            //intercept combobox res changed and update width/height value
+            this.Ref_combobox_Resolution.connect('changed',
+                Lang.bind(this, function(self) {
+                    var activeRes = self.active;
```

Comment:
I wonder if we should add new code with `let` variables because of the clearer scope.